### PR TITLE
feat(wasm-storage): add unit test for msg server

### PR DESCRIPTION
**Pull request by JeancarloBarrios**
Added test for msg server and querier

### CODE REVIEW

File: x/wasm-storage/keeper/msg_server.go [hunk 1]
```diff
@@ -76,14 +82,15 @@ func (k msgServer) StoreOverlayWasm(goCtx context.Context, msg *types.MsgStoreOv
...
+func unzipWasm(wasm []byte) ([]byte, error) {
 	var unzipped []byte
 	var err error
-	if ioutils.IsGzip(wasm) {
-		unzipped, err = ioutils.Uncompress(wasm, types.MaxWasmSize)
-		if err != nil {
-			panic(err)
-		}
+	if !ioutils.IsGzip(wasm) {
+		return nil, fmt.Errorf("wasm is not gzip compressed")
```
Comment (JeancarloBarrios) [inline x/wasm-storage/keeper/msg_server.go:89]: Change this so it returns an error when the wasm was not compressed, to fix the behavior "non gzip wasm files would be hash as nil"

Comment (hacheigriega) [reply x/wasm-storage/keeper/msg_server.go:89]: nice catch - tyty

**Review by hacheigriega — APPROVED** [review]
LGTM. Thank you for writing all these tests.

File: x/wasm-storage/keeper/msg_server.go (hunk 1 above)
Comment (hacheigriega) [reply x/wasm-storage/keeper/msg_server.go:89]: nice catch.. tyty

**Pull request merged**